doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -85,13 +85,14 @@ SUIT DFU for nRF54H20
 Nordic Secure Immutable Bootloader (NSIB, B0, or B0n)
 -----------------------------------------------------
 
-Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.
-The ``CONFIG_SECURE_BOOT_DEBUG`` Kconfig option has been removed.
-To disable logging in B0 or B0n, set the :kconfig:option:`CONFIG_LOG` option to ``n``.
-To send logs over RTT instead of UART, apply the following settings:
+.. toggle::
+   Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.
```

Comment:
Moving this text to under toggle will result in text not showing up... It only displays bullet points it seems :)

Reply:
I did that because it seems that migration guide areas are meant to be like this (most are this way). Or what do you mean exactly, is there something wrong with this particular instance? There might have been a missing newline. I fixed that.

Reply:
My bad, it was the missing line, I tested it, but never got back to you.